<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Folder &amp; File Creator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="filename_label">
         <property name="text">
          <string>File Name </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="file_name_line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="directory_label">
         <property name="text">
          <string>Directory  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="directory_line"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="content_list_widget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="chose_dir_button">
       <property name="text">
        <string>Chose a Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="create_dir_button">
       <property name="text">
        <string>Create Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="create_file_button">
       <property name="text">
        <string>Create File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_contenct_button">
       <property name="text">
        <string>Folder Content</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
